docs/section6/Rule6-5.md

```diff
@@ -16,7 +16,7 @@
 **Function Call:**  
 
   - compare_schedules()
-  - normalize_space_schedules()
+  - normalize_interior_lighting_schedules()
 
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
```

Comment:
Logic below starts with "for each building_segment"... We should start it with "For each building".

Reply:
Having looked at all the relevant RDSs and functions yesterday, I think we do need to account for control_fraction as part of normalize_interior_lighting_schedules. For the proposed design, each space may have multiple interior lighting objects, and while all of them would be subject to the same adjustment based on Table G3.7 (because they are all associated with the same space of given type), each lighting object may have different occupancy_control_type, so some of the schedules may be subject to additional reduction per Table G3.7 footnote b and some won't. @weilixu and @supriyagoel let me know if you agree.

Reply:
@mkarpman You are correct. Because we are summing up the normalized lighting power in a space, so we need the control credit for each interior lighting data group.

In this case, we can add an additional argument to flag whether the algorithm shall apply the control credit or not.?